src/main.rs

```diff
@@ -50,6 +50,7 @@ use std::env;
 use rocket::config::RocketConfig;
 
 fn main() {
+    /*
```

Comment:
should kill this

Reply:
Wait, why is this still here? dammit, git...